$MirOS$

Note: first chunk must look: $CC $CFLAGS '-DCPP="@PREFIX@/bin/9cpp"' lex.c

--- utils/cc/mkfile.orig	Fri Mar 31 00:30:56 2006
+++ utils/cc/mkfile	Fri Mar 31 00:31:15 2006
@@ -32,4 +32,4 @@ CFLAGS=	$CFLAGS -I../include
 mac.$O: macbody
 
 lex.$O:	lex.c
-	$CC $CFLAGS '-DCPP="/bin/cpp"' lex.c
+	$CC $CFLAGS '-DCPP="@PREFIX@/bin/9cpp"' lex.c
